dataland-keycloak-adapter/src/main/kotlin/org/dataland/keycloakAdapter/auth/DatalandRealmRole.kt

```diff
@@ -5,6 +5,6 @@ package org.dataland.keycloakAdapter.auth
  * Datalandsecurity realm. Can be used in conjunction with the
  * Authentication.hasRole extension function to check if a user has a certain role
  */
-enum class DatalandRealmRoles {
+enum class DatalandRealmRole {
```

Comment:
Does the renaming have a specific reason? I personally prefer the plural version, probably just because I'm used to it since I can not come up with a different reason. The question "What does the type contain?" would be better answered by the plural version but the question "What is it?" is better answered with the singular version. However, in the code we seem to do both. So do what you want. Sorry for the monologue :)

Reply:
No worries. From my point of view each enum constant only referrs to a single Role and not multiple Roles. Thats why I chose to use singulars.